recipes/pcapplusplus/all/conanfile.py

```diff
@@ -1,5 +1,9 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools import files
+from conan.tools.microsoft import MSBuild, is_msvc
+from conan.tools.microsoft.visual import vs_ide_version
```

Comment:
@uilianries linter is complaining about this, is this a linter false positive or is there a better syntax for this?

Reply:
The linter is correct, you can't import submodels, 3 levels under tools. It's documented here: https://docs.conan.io/en/latest/reference/conanfile/tools.html

You can't use `vs_idle_version`